frontend/src/data/TestSkills.tsx

```diff
@@ -0,0 +1,34 @@
+import { Item } from '../types/ItemTypes';
+
+const Skills: Item[] = [
+    {
+        id: 1,
+        name: 'Action-Oriented',
+    },
+    {
+        id: 2,
+        name: 'Aggressive',
+    },
+    {
+        id: 3,
+        name: 'Agile',
+    },
+    {
+        id: 4,
+        name: "Aggro Crab's Blessing",
+    },
+];
+
+const mappedSkills = new Map<number, Item>();
+Skills.forEach((skill) => mappedSkills.set(skill.id, skill));
+
+function skillFromId(id: number): Item {
+    const skill = mappedSkills.get(id);
+    if (!skill) {
+        // Sanity check - This should never be thrown
```

Comment:
FYR may need real checks in here in case someone messes with the source config files

Reply:
This was removed in #22 :)